spec/models/miq_report_spec.rb

```diff
@@ -491,6 +491,23 @@ def invoke_ae
       end
     end
 
+    context "chargeback report" do
+      let(:report) do
+        MiqReport.new(
+            :title   => "chargeback_report",
+            :db      => "Chargeback",
+            :cols    => %w(start_date display_range entity_namecpu_cost cpu_metric cpu_used_cost
+                           cpu_used_metric fixed_compute_1_cost fixed_compute_2_cost fixed_cost
+                           memory_cost memory_metric memory_used_cost memory_used_metric total_cost),
+            :include => {})
+      end
+      let(:ems) { FactoryGirl.create(:ems_vmware, :zone => @server.zone) }
+
+      it "runs report" do
+        report.generate_table(:userid => "admin")
```

Comment:
> anything about the NotImplementedError? :/

Nope, expecting _not_ `NotImplementError` (double negative) is actually a false positive as other errors that could be raised (by Ruby, for instance) could make that test pass. 

I'm checking that above spec, I'm skeptical that it actually does what it says it does.